scripts/jquery.topics.js

```diff
@@ -617,3 +621,27 @@ function loadSavedFilterOptions(filter_options) {
 		$(".keepers_filter_rule_fieldset").show();
 	}
 }
+
+function showStateKeeperIconDescription() {
+	$(".topic_data>i.fa-arrow-circle-up").attr("title", "Сидируется, отсутствует в отчетах");
+	$(".topic_data>i.fa-upload").attr("title", "Сидируется, присутствует в отчетах");
+	$(".topic_data>i.fa-arrow-up").attr("title", "Не сидируется, присутствует в отчетах");
+	$(".topic_data>i.fa-arrow-down").attr("title", "Не сидируется, отсутствует в отчетах");
+	$(".topic_data>i.text-self").attr("title", $(".topic_data>i.text-self").attr("title") + ", Я");
+	$(".topic_data>i.text-success").attr("title", $(".topic_data>i.text-success").attr("title") + ", раздается");
+	$(".topic_data>i.text-danger").attr("title", $(".topic_data>i.text-danger").attr("title") + ", скачивается");
+	$(".keeper.text-self").attr("title", "Я");
+	$(".keeper.text-success").attr("title", "Раздается");
+	$(".keeper.text-danger").attr("title", "Скачивается");
+}
+
+function showStateTorrentClientIconDescription() {
+	$(".topic_data>label>i.fa-arrow-up").attr("title", "Сидируется");
+	$(".topic_data>label>i.fa-circle").attr("title", "Отсутствует в клиентах");
+	$(".topic_data>label>i.fa-arrow-down").attr("title", "Скачивается");
+	$(".topic_data>label>i.fa-pause").attr("title", "На паузе");
+	$(".topic_data>label>i.fa-times").attr("title", "С ошибкой в клиенте");
+	$(".topic_data>label>i.text-danger").attr("title", $(".topic_data>label>i.text-danger").attr("title") + ", данных о средних сидах нет даже за половину периода");
+	$(".topic_data>label>i.text-warning").attr("title", $(".topic_data>label>i.text-warning").attr("title") + ", данные о средних сидах есть более чем за половину периода");
+	$(".topic_data>label>i.text-success").attr("title", $(".topic_data>label>i.text-success").attr("title") + ", данные о средних сидах есть за весь период");
```

Comment:
Say, we have a modest 1000 rows. Or 2000. Then, just to add title to each of them, we create 20000/40000 jQuery objects, use nested selectors…

IMHO, far more efficient here is to add those titles in PHP side.

Reply:
Agree, new PR will be opened by @Evgen-BioNik with php implementation.